src/common/components/storage/StorageDetail.tsx

```diff
@@ -94,6 +158,26 @@ const DisksTable = ({
         { title: disk.serial, props: { 'data-testid': 'disk-serial' } },
         { title: disk.model, props: { 'data-testid': 'disk-model' } },
         { title: disk.wwn, props: { 'data-testid': 'disk-wwn' } },
+        (isDiskFormattable(host, disk.id) || isInstallationDisk(disk)) && {
+          title: (
+            <Tooltip
+              hidden={!isInstallationDisk(disk)}
+              content={'Installation disks are always being formatted'}
+            >
+              <Checkbox
+                id={`select-formatted}`}
+                isChecked={isInstallationDisk(disk) || !isDiskSkipFormatting(host, disk.id)}
+                onChange={(doFormatDisk: boolean) =>
```

Comment:
The checkboxes should not be interactive if the prop `updateDiskSkipFormatting` is not defined.

Reply:
Modified